src/ngraph/runtime/cpu/cpu_builder_optimized.hpp

```diff
@@ -0,0 +1,101 @@
+//*****************************************************************************
```

Comment:
No real need to change but just for consideration. The file name could be cpu_builder_size_optimized because optimized is a loaded word.